From 8e3e45c4a4ae63dd19d1bf05d98897e6e0d28b39 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 20 Jul 2022 14:53:51 +0300
Subject: [PATCH] [SW-95811] gaudi2: export security_enable_jtag_if

security_enable_jtag_if() function from arcboot
to be used to enable JTAG when rma_enabled.
So updated export.elf for gaudi2 with arcboot
exported list modified to include this symbol.
This would avoid duplicate implementation of
same function in ppboot.

New exports.elf for gaudi2 generated on relevant gaudi2
arcboot rom branch i.e. 'gaudi2_secureboot_rom_tapeout_release6'.
(https://gerrit.habana-labs.com/#/c/234408/)

But as old gaudi2 version is not maintained, so updated the
symbol on top of latest master.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I3860b207bed63231153b690557a7d662cd27284d
---

diff --git a/arch/gaudi2/arc/arcboot_rom.txt b/arch/gaudi2/arc/arcboot_rom.txt
index 92277c6..f0440a4 100644
--- a/arch/gaudi2/arc/arcboot_rom.txt
+++ b/arch/gaudi2/arc/arcboot_rom.txt
@@ -4,3 +4,4 @@
 secureboot_verify_image
 dma_copy_image
 spi_flash_readl
+security_enable_jtag_if
